### ISSUE TITLE
Reproducibility training metrics

### ISSUE THREAD

**Issue by JohnnMa** [monkek123King/CLD#7]
Thank you to the authors for publishing their work. I used the published code with the PrismLayersPro dataset to train 25K steps, tested with 5% of the data, and evaluated using eval.py. The metrics differed significantly from those in the paper. What could be the reason for this? Is there any extra trick in the training code?

My running results:

Layer PSNR: 20.514382958762624
Layer SSIM: 0.6130648287368728
Layer FID: 18.08108132572044
Mask IoU: 0.6630245207805613
Mask Precision: 0.736122766617873
Mask Recall: 0.803050007738107
Mask F1: 0.7638011187804984
Composite PSNR: 19.401147516421947
Composite SSIM: 0.8200624062342233
Composite FID: 17.974414293431117

**Comment by monkek123King** [comment]
Thank you for reproducing our work.

I suspect that your max_steps was set to 25K. The main reason is the difference between paper iterations and the code’s max_steps. Our training uses gradient accumulation with accum_steps = 4. The 25K iterations reported in the paper correspond to 25K effective parameter updates, which means max_steps should be set to 25K × 4 = 100K in the code.

**Comment by JohnnMa** [comment]
Thanks for your reply.  It's help.
